docs/en/setup/backend/backend-ariflow-monitoring.md

```diff
@@ -0,0 +1,56 @@
+# Apache Airflow monitoring
+## Airflow  server performance from 'OpenTelemetry'
+SkyWalking leverages Airflow for collecting metrics data. It leverages OpenTelemetry Collector to transfer the metrics to
+[OpenTelemetry receiver](opentelemetry-receiver.md) and into the [Meter System](./../../concepts-and-designs/meter.md).
+
+### Data flow
+
+1. OpenTelemetry Collector fetches metrics from Airflow via OTEL and pushes metrics to SkyWalking OAP Server via OpenTelemetry exporter.
```

Comment:
How to fetch through OTEL? OTEL protocol is a push style.

Could you clarify this?